axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -676,14 +693,29 @@
         "address": "axelar1jjjr3tqs0nzjv3y9fg4xvzkww50jq06a9qp77r8kzmqyla97556sxx7702"
       },
       "AxelarnetGateway": {
-        "codeId": 661,
-        "address": "axelar1yvfcrdke7fasxfaxx2r706h7h85rnk3w68cc5f4fkmafz5j755ssl8h9p0"
+        "codeId": 844,
+        "address": "axelar1wvms3cy5hxrgl7uxhkz7yth4qzqum6aaccwkmvafq8z0mgdfxr8qrnvw0k",
+        "nexus": "axelar17h8uk4ct0mdv9mgkuxszt4gp2htpfr08mge20r",
+        "axelar": {
+          "storeInstantiateProposalId": "59",
+          "codeId": 845
+        },
+        "storeCodeProposalCodeHash": "c7286d0f59276b794641bdfbb4f96fafcee3553b67f3397d662a4683968f525b",
+        "storeCodeProposalId": "60",
+        "lastUploadedCodeId": 845
       },
       "InterchainTokenService": {
         "adminAddress": "axelar1zlr7e5qf3sz7yf890rkh9tcnu87234k6k7ytd9",
         "governanceAddress": "axelar1zlr7e5qf3sz7yf890rkh9tcnu87234k6k7ytd9",
         "codeId": 738,
-        "address": "axelar10jzzmv5m7da7dn2xsfac0yqe7zamy34uedx3e28laq0p6f3f8dzqp649fp"
+        "address": "axelar10jzzmv5m7da7dn2xsfac0yqe7zamy34uedx3e28laq0p6f3f8dzqp649fp",
```

Comment:
the address didn't change, doesn't look like this was redeployed?

Reply:
Changed the address manually. The contract is indeed deployed, not sure why the script didn't change the file afterwards
```
axelard query wasm contract-state raw axelar157hl7gpuknjmhtac2qnphuazv2yerfagva7lsu9vuj2pgn32z22qa26dk4 636F6E74726163745F696E666F --node http://devnet-amplifier.axelar.dev:26657 -o json | jq -r '.data' | base64 -d
{"contract":"interchain-token-service","version":"1.0.0"}`
```